tools/db_bench_tool.cc

```diff
@@ -604,7 +603,7 @@ DEFINE_bool(use_stderr_info_logger, false,
 
 DEFINE_bool(use_blob_db, false, "Whether to use BlobDB. ");
 
-enum rocksdb::CompressionType StringToCompressionType(const char* ctype) {
+static enum rocksdb::CompressionType StringToCompressionType(const char* ctype) {
```

Comment:
i guess to reduce the line count :p. functions are currently interleaved with the argument definitions so we'd need a lot of "namespace {" and "}  // anonymous namespace" lines to keep them as they were. plus most of the other interleaved functions were already static (unnecessarily so until now).